mdtraj/core/selection.py

```diff
@@ -43,6 +42,12 @@
     quotedString,
 )
 
+try:
+    from ast import unparse
+except ImportError:
+    from astunparse import unparse
```

Comment:
Any reason the stdlib import wouldn't work? We could just drop the `astunparse` dependency altogether